secprofile/oci-profile.go

```diff
@@ -111,8 +111,6 @@ func allowSyscallWithArgs(seccompProfile *specs.LinuxSeccomp, syscallName types.
 
 	syscallNameToAllowStr := string(syscallName)
 
-	logrus.Errorf("Allowing syscall: %s", syscallNameToAllowStr)
```

Comment:
why remove this?  It may be helpful as a debug level

Reply:
I felt like this info wasn't very useful as its pretty much repeats what the function is doing. Will undo that and switch it to debug if you think we should keep it.

Reply:
ok.  Let's leave this out and create a tracking issue for logging in libentitlement.  If there is enough demand, we can add it over the core codebase all at once

Reply:
Sounds good! Issue here: https://github.com/docker/libentitlement/issues/46